### PR TITLE
#7/unsubscribe from all channels functionality

### PR DESCRIPTION
**Pull request by dimglyn**
Resolves issue #7 
I added the new AllPlatformsSettings so I leave SettingsContainer stateless as it already was.

### CODE REVIEW

File: app/components/AllPlatformsSettings.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+const { SUPPORTED_PLATFORMS } = require('../constants');
+const PlatformSetting = require('./PlatformSetting');
+
+const AllPlatformsSettings = React.createClass({
+    onClickHandler() {
+        SUPPORTED_PLATFORMS.forEach(platform => {
+            store.disablePlatform(platform);
+            ga('send', 'event', 'Platform Setting', 'UnCheck', platform);
+        });
+        this.forceUpdate();s
```
Comment (nishanthvijayan) [inline app/components/AllPlatformsSettings.js]: What's the s at the end?

File: css/chrome.css [hunk 2]
```diff
@@ -163,6 +163,16 @@ header > h3 {
 .green-text {
   color: #4caf50;
 }
+
+.button-unsubscribe {
+  background-color: #f44336;
```
Comment (nishanthvijayan) [inline css/chrome.css:168]: Change cursor to pointer. Looks nicer.

File: css/firefox.css [hunk 3]
```diff
@@ -159,6 +159,16 @@ header > h3 {
 .green-text {
   color: #4caf50;
 }
+
+.button-unsubscribe {
+  background-color: #f44336;
```
Comment (nishanthvijayan) [inline css/firefox.css:164]: Same

**Review by nishanthvijayan — CHANGES_REQUESTED** [review]

File: app/components/AllPlatformsSettings.js (hunk 1 above)
Comment (dimglyn) [reply app/components/AllPlatformsSettings.js]: Apparently I wanted to Ctrl+S but my Ctrl is kinda broken

**Comment by dimglyn** [comment]
Latest commit should resolve all mentioned above

**Review by nishanthvijayan — APPROVED** [review]

**Pull request merged**